### PR TITLE
Support Node.js comparisons

### PR DESCRIPTION
**Pull request by Mathspy**
Resolves: https://github.com/browserslist/browserslist/issues/293

### CODE REVIEW

File: test/node.test.js [hunk 1]
```diff
@@ -39,3 +39,146 @@ it('supports current node version', () => {
...
+    'node 0.5.0',
+    'node 0.4.0',
+    'node 0.3.0',
+    'node 0.2.0',
+    'node 0.12.0',
+    'node 0.11.0',
+    'node 0.10.0'
+  ])
+
+  expect(browserslist('node >= 9')).toEqual(expect.arrayContaining([
```
Comment (Mathspy) [inline test/node.test.js]: Note, greater than test use `expect.arrayContaining` to avoid breaking in the future once Node is updated

**Comment by Mathspy** [comment]
Not sure I understand why `Node.js: node` fails  ¯\_(ツ)_/¯

**Comment by ai** [comment]
@Mathspy I pushed fix to the master 6af1110. Can you rebase the branch?

**Comment by Mathspy** [comment]
@ai Done! :heart:

**Comment by ai** [comment]
Amazing. I will release it tomorrow.

**Comment by Mathspy** [comment]
Thanks for the quick review :tada:

**Comment by ai** [comment]
Thanks. Released in 4.5.

**Pull request merged**